### PR TITLE
"How to accept Monero" - Portuguese Translation

### PR DESCRIPTION
**Pull request by 0xyw**

### CODE REVIEW

**Comment by lh1008** [comment]
Review assigned to @botoserelepe :+1:

**Review by saxocat — DISMISSED** [review]
Hi,
I didn't find any errors.
You're free to publish!

Merry Christmas and a Happy New Year!

**Review by botoserelepe — CHANGES_REQUESTED** [review]
@lh1008 review done! Sorry for the late answer, I had forgotten about it after the Xmas holidays! My fault.
@0xyw Awesome translation! I only did some minor suggestions and corrections.
Some of the links of this article might exist in Portuguese, so I think it is useful to check them before publishing.

**Comment by saxocat** [comment]
Hey!
If needed I can also write the european portuguese (pt_PT) version since some of the semantics change.
Let me know if that would help!

File: monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Com um processador de pagamento de código aberto, você pode desenvolver sua própria solução usando a API de sua exchange preferida com monero-wallet-rpc. Isso está além do escopo deste tutorial, mas você pode encontrar mais informações em [getmonero.org/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/pt-br/resources/developer-guides/wallet-rpc.html).
+
+### _Saiba Mais_
+
+- Como aceitar Monero com carteiras oficiais Monero: [getmonero.org/get-started/accepting](https://www.getmonero.org/pt-br/get-started/accepting/index.html)
+- [FAQs dos Comerciantes Monero](https://www.monerooutreach.org/merchants/monero-merchant-faqs.html)
+- Explicando Assinaturas Múltiplas de Monero: [hackernoon.com/monero-multisignatures-explained...](https://hackernoon.com/monero-multisignatures-explained-46b247b098a7)
+- Riscos Potenciais de Aceitar Transações de Zero Confirmação: [reddit.com/r/Monero/.../potential_risks_of_accepting_zero_confirmation](https://www.reddit.com/r/Monero/comments/7s937y/potential_risks_of_accepting_zero_confirmation/)
+
+Ilustrado por:
```
[the remark this answers is not in the document]
Comment (0xyw) [reply monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md:85]: there is no info about the illustrator.
https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/en/merchants/how_to_accept_monero_en.md

**Comment by 0xyw** [comment]
> Hey!
> If needed I can also write the european portuguese (pt_PT) version since some of the semantics change.
> Let me know if that would help!

tens telegram?

**Comment by 0xyw** [comment]
nice review. I think it's ready for merge @lh1008 @erciccione

File: monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# Como aceitar Monero
+
+Se você consegue instalar um aplicativo, você estará pronto para aceitar Monero (XMR) no seu negócio. Esse guia explica as formas mais comuns que comerciantes usam para aceitar Monero e como começar sem dor de cabeça a se desenvolver e a crescer o seu negócio.
+
+Primeiramente não há um tipo especial de conta comercial Monero - você apenas precisa de uma carteira e um endereço normal para começar. É recomendado que você crie um endereço separado para transações comerciais pela mesma razão de que é cômodo e seguro ter contas pessoais e de negócios separadas. Você pode usar o mesmo software para a carteira, mas ter endereços dedicados para negócios irá ajudar manter as contas organizadas.
+
+Se você ainda não tem uma carteira/endereço Monero então é melhor criar uma primeiro. As carteiras Monero GUI/CLI oficiais de [getmonero.org/downloads](https://www.getmonero.org/pt-br/downloads/index.html) são sempre recomendadas. Além disso, esse guia irá indicar carteiras mobile e você pode achar informação sobre elas na página de [Configuração da Carteira Monero](https://www.monerooutreach.org/stories/monero-wallet-quickstart.html).
```
[the remark this answers is not in the document]
Comment (0xyw) [reply monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md:7]: I did not find it.

**Review by botoserelepe — DISMISSED** [review]

File: monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (0xyw) [reply monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md:7]: true, but the article isn't available on the official website

```suggestion
Se você ainda não tem uma carteira/endereço Monero então é melhor criar uma primeiro. As carteiras Monero GUI/CLI oficiais de [getmonero.org/downloads](https://www.getmonero.org/pt-br/downloads/index.html) são sempre recomendadas. Além disso, esse guia irá indicar carteiras mobile e você pode achar informação sobre elas na página de [Configuração da Carteira Monero](https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md). 
```

File: monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+3. **Pool de Transações** - Dentro de um segundo, a transação é anunciada para a rede Monero e colocada no pool de transações, também chamada de mempool (pool de memória). Você pode verificar a existência de sua transação no mempool, um exemplo pode ser visto aqui: xmrchain.net/txpool
+4. **Confirmação** - Dentro de 4 minutos, a transação irá geralmente estar em um bloco da blockchain e você pode ter confiança de que ela é válida. O pagamento irá aparecer em sua carteira como fundos bloqueados. Sua transação será confirmada por mais 9 blocos antes de ser desbloqueada e virtualmente impossível de se reverter.
+    
+Pode haver um lag na verificação de novas transações, geralmente 90 segundos. Portanto, dependendo de quando sua carteira verifica se há novas transações e com que rapidez sua transação entra em blocos minerados, o tempo real esperado antes que ela apareça em sua carteira é de 4 a 12 minutos. Para uma análise mais aprofundada do processo de transação do Monero e da duração média, visite [monero.how/how-long-do-monero-transactions-take](https://www.monero.how/how-long-do-monero-transactions-take).
```
[the remark this answers is not in the document]
Comment (0xyw) [reply monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md:18]:
```suggestion
Pode haver um atraso na verificação de novas transações, geralmente 90 segundos. Portanto, dependendo de quando sua carteira verifica se há novas transações e com que rapidez sua transação entra em blocos minerados, o tempo real esperado antes que ela apareça em sua carteira é de 4 a 12 minutos. Para uma análise mais aprofundada do processo de transação do Monero e da duração média, visite [monero.how/how-long-do-monero-transactions-take](https://www.monero.how/how-long-do-monero-transactions-take).
```

**Comment by saxocat** [comment]
> > Hey!
> > If needed I can also write the european portuguese (pt_PT) version since some of the semantics change.
> > Let me know if that would help!
> 
> tens telegram?

Tenho!
O meu username é saxocat

File: monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+A maioria dos proprietários de empresas está familiarizada com processadores de pagamento de crédito/débito, terceiros que alugam o terminal de você, o colocam em sua rede e cobram taxas de transação por mês. Um processador de pagamentos Monero pode funcionar assim ou você pode ser o seu próprio com um pequeno software e não pagar nenhuma taxa de transação.
+
+Como o software Monero é de código aberto, você pode criar seu próprio gateway de pagamento do zero (muito legal isso, né?), mas Monero Integrations já fez o trabalho para você e criou bibliotecas e plug-ins que fazem todo o trabalho pesado em [github.com/monero-integrations](https://github.com/monero-integrations).
+
+Como alternativa, um processador de pagamento Monero de terceiros terá softwares, aplicativos e interfaces que simplificam a configuração. Eles também terão interfaces e ferramentas para você gerenciar sua conta e suporte técnico/ao cliente. Consulte nosso Guia do Processador de Pagamentos Monero para obter ajuda para decidir qual configuração é a certa para você.
```
[the remark this answers is not in the document]
Comment (0xyw) [reply monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt_br.md]: my fault on making this suggestion outdated @lh1008 

```suggestion
Como alternativa, um processador de pagamento Monero de terceiros terá softwares, aplicativos e interfaces que simplificam a configuração. Eles também terão interfaces e ferramentas para você gerenciar sua conta e suporte técnico ao cliente. Consulte nosso guia de [Processadores de Pagamentos Monero](https://www.monerooutreach.org/merchants/monero-payment-processor-guide.html) para obter ajuda para decidir qual configuração é a certa para você.
```

**Comment by 0xyw** [comment]
I think it's ok now.

**Review by botoserelepe — APPROVED** [review]

**Review by lh1008 — APPROVED** [review]
Thank you @0xyw, @saxocat, and @botoserelepe, great work. This should be ready for merge then.

**Pull request merged**